lib/cli/optimizeImages/optimizeImages.ts

```diff
@@ -0,0 +1,58 @@
+import { buntstift } from 'buntstift';
+import { ensureFile } from 'fs-extra';
+import globby from 'globby';
+import { optimizeJpg } from './optimizeJpg';
+import { optimizePng } from './optimizePng';
+import { optimizeSvg } from './optimizeSvg';
+import os from 'os';
+import path from 'path';
+import PQueue from 'p-queue';
+
+const optimizerForExtension = {
+  jpg: optimizeJpg,
+  jpeg: optimizeJpg,
+  png: optimizePng,
+  svg: optimizeSvg
+};
+
+type SupportedExtensionNames = keyof typeof optimizerForExtension;
+const supportedExtensions = Object.keys(optimizerForExtension);
+
+const optimizeImages = async function ({
+  source,
+  destination,
+  verbose
+}: {
+  source: string;
+  destination: string;
+  verbose: boolean;
+}): Promise<void> {
+  const queue = new PQueue({ concurrency: os.cpus().length });
+
+  const imagePaths = await globby(source, {
+    expandDirectories: {
+      extensions: supportedExtensions
+    }
+  });
+
+  for (const sourcePath of imagePaths) {
```

Comment:
Shouldn't this be `imagePath`?

```suggestion
  for (const imagePath of imagePaths) {
```